tests/monitoring/component_monitoring.go

```diff
@@ -112,44 +113,109 @@ var _ = Describe("[sig-monitoring]Component Monitoring", Serial, decorators.SigM
 
 	Context("Up metrics", func() {
 		BeforeEach(func() {
+			By("Backing up the operator, controller, api and handler scales")
 			scales = libmonitoring.NewScaling(virtClient, []string{virtOperator.deploymentName, virtController.deploymentName, virtApi.deploymentName})
 			scales.UpdateScale(virtOperator.deploymentName, int32(0))
 
+			By("Reducing the alert pending time")
 			libmonitoring.ReduceAlertPendingTime(virtClient)
 		})
 
 		AfterEach(func() {
-			scales.RestoreAllScales()
-			waitUntilComponentsAlertsDoNotExist(virtClient)
+			By("Restoring the operator")
+			oldVirtOperatorReplicas := scales.GetScale(virtOperator.deploymentName)
+
+			for i := int32(1); i <= oldVirtOperatorReplicas; i++ {
```

Comment:
`for i := range oldVirtOperatorReplicas`

Reply:
changed, but need to add 1, to every iteration, so, eh